### ISSUE TITLE
View reactions is broken

### ISSUE THREAD

**Issue by ParagDoke** [hoehermann/purple-gowhatsapp#177]
Thank you for an excellent plugin for WhatsApp !

Disclaimer: Pidgin novice here. Very likely I am missing something.

Using:
Pidgin 2.14.12 (libpurple 2.14.12)
2fc862e230a7d64799e614a49ee49ef9d54f89a9
On Windchill 11 64 bit OS

For this plugin:
Author: Hermann Hoehne <hoehermann@gmx.de>
Version: 1.13.0
Website: https://github.com/hoehermann/purple-gowhatsapp
ID String: prpl-hehoe-whatsmeow
Loadable: Yes
Loaded: Yes

When I receive reactions on my message, this is what is rendered:
![image](https://github.com/hoehermann/purple-gowhatsapp/assets/934058/a7886cc7-2c26-4295-8a80-30a0ef6803a6)
I see names ... have blanked them deliberately. So that is not an issue.

I see on the default page of this repo that reactions should work. And there are some settings too. Maybe I missed it ... should we add a link to how settings should be configured (a link to Pidgin documentation ... I understand that it may not be for this plugin).

TLDR: How does a novice get reactions to work?

**Comment by EionRobb** [comment]
Windows versions of Pidgin don't support emoji at this stage. You'll need to use a smiley theme (eg https://github.com/niclashoyer/pidgin-emojione) to let Pidgin render them. 

This isn't a bug with the WhatsApp plugin, it affects all plugins and protocols.

**Comment by ParagDoke** [comment]
Thank you @EionRobb ! Will close this issue.
